app/router/router.go

```diff
@@ -0,0 +1,116 @@
+package router
```

Comment:
Perhaps this can live in the lbrynet package — I was going to delegate all lbrynet server-related communications to that module eventually anyway.